### PR TITLE
[OBMIN-108] Change CSS to Style-components (file ProductPhotoCarousel), updated some files. (merge 108 & 109)

### PR DESCRIPTION
**Pull request by cloudmenson**

### CODE REVIEW

**Review by Wolshebnik — COMMENTED** [review]
good

File: src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/NoPhoto/NoPhoto.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import noPhotos from '../../../../assets/img/showAdv/noPhoto.svg';
+
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/NoPhoto/NoPhoto.jsx:2]: absolute path pls

File: src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/index.jsx [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 import SliderOrNot from '../SliderOrNot';
-import NoPhoto from './NoPhoto';
 
-import './ProductPhotoCarousel.scss';
+import {
+  CarouselBox,
+  ProductPhotoSlideBig,
+  SliderPosition,
+  Image,
+} from './styles';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/index.jsx:12]: sort by length

File: src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/index.jsx [hunk 3]
```diff
@@ -25,27 +30,27 @@ const ProductPhotoCarousel = ({ photos }) => {
...
   if (!state.photos.length) {
-    noArr = <NoPhoto noPhoto="bigNoPhoto" noPhotoImg="bigNoPhotoImg" />;
+    noArr = <NoPhoto noPhoto />;
   } else {
     noArr = (
-      <img
+      <Image
         src={`data:image/jpeg;base64,${state.bigPhoto.resource}`}
         alt="activeSlide"
       />
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/index.jsx:39]: sort

File: src/main/resources/react/src/pages/ProductPage/ProductPostData/index.jsx [hunk 4]
```diff
@@ -4,7 +4,20 @@ import { getLang } from 'store/auth/slice';
...
+  ButtonContainer,
+  PostDataDescription,
+  PostDataBoxContainer,
+  PostDataDescriptionOl,
+  PostDataDescriptionUl,
+  PostDataDescriptionText,
+  PostDataDescriptionSpan,
+  PostDataDescriptionOlItem,
+  PostDataDescriptionUlItem,
+} from './styles';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ProductPage/ProductPostData/index.jsx:21]: in the future I want to see 
import * as Styles from './styles'

File: src/main/resources/react/src/assets/icons/index.js [hunk 5]
```diff
@@ -1,31 +1,14 @@
...
+export { ReactComponent as HeartSvg } from './heart.svg';
+export { ReactComponent as PhoneSvg } from './phone.svg';
+export { ReactComponent as LogoutSvg } from './logout.svg';
+export { ReactComponent as SearchSvg } from './search.svg';
+export { ReactComponent as GoogleSvg } from './google.svg';
+export { ReactComponent as ProfileSvg } from './profile.svg';
+export { ReactComponent as ActivitySvg } from './activity.svg';
+export { ReactComponent as LocationSvg } from './location.svg';
+export { ReactComponent as SettingsSvg } from './settings.svg';
+export { ReactComponent as QuestionMarkSvg } from './question-mark.svg';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/assets/icons/index.js:14]: good

**Review by smilaya — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**